src/controllers/TalksController.php

```diff
@@ -632,6 +653,21 @@ public function getTalkMapper($db, $request)
         return $this->talk_mapper;
     }
 
+    public function setEventMapper(EventMapper $event_mapper)
```

Comment:
Just an idea as I'm seeing a lot of setters and getters sprouting over the next commits: Can't we use Factory-Classes for getting the mappers? Either the default mapper is instantiated or you can call a static setter and then the set mapper will be returned. Basically moving the setMapper/getMapper into its own class.